.github/workflows/release-build.yml

```
@@ -17,11 +17,21 @@ env:
 
 jobs:
   release-build:
-    name: Build ${{ matrix.target }}
+    name: Build ${{ matrix.backend }} / ${{ matrix.os_arch }}
```

actionlint (1.7.12)

[error] property "os_arch" is not defined in object type {backend: {feature: string; manifest: string; name: string}; platform: {os: string; os_arch: string; target: string; use_cross: bool}}

(expression)